march_data_collector/src/march_data_collector/data_collector_node.py

```diff
@@ -63,16 +71,18 @@ def imu_callback(self, data):
             transform.transform.rotation.z = imu_rotation[2]
             transform.transform.rotation.w = imu_rotation[3]
 
-            self._imu_broadcaster.sendTransform(transform)
+            self.imu_broadcaster.sendTransform(transform)
```

Comment:
`self.imu_broadcaster` is not a defined value
```suggestion
            self._imu_broadcaster.sendTransform(transform)
```